chart-testing: Add version 3.7.1

stevehipwell opened the pull request:
<!-- Provide a general summary of your changes in the title above -->
This PR adds the Helm [chart-testing](https://github.com/helm/chart-testing) CLI.

<!--
  By opening this PR you confirm that you have searched for similar issues/PRs here already.
  Failing to do so will most likely result in closing of this PR without any explanation.
  It is also mandatory to open a relevant issue (either Package Request or Bug Report) for
  discussion with the maintainers, before creating any new PR.
  Read the contributing guide first to save both your and our time.
-->

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

stevehipwell commented:
/verify

rashil2000 reviewed (changes requested):
- Arm64 variant
- `chart-testing.json` -> `helm-chart-tester.json`

stevehipwell commented:
@rashil2000 I've added the `arm64` version and renamed the manifest to `helm-chart-testing` (it would be incorrect to rename it to `helm-chart-tester` as that's not it's name).

rashil2000 commented on bucket/helm-chart-testing.json:
```diff
@@ -0,0 +1,35 @@
+        }
+    },
+    "post_install": [
+        "New-Item -ItemType Directory -Path \"~/.ct\" -Force | Out-Null",
```
Is this post_install necessary? We don't want to make changes, that aren't strictly required for the functioning of the app, outside of the installation folder.

stevehipwell replied:
Yes, it's 100% required or else `ct lint` and `ct lint-and-install` will not work.

rashil2000 replied:
According to the docs:

> Note that linting requires config file for [yamllint](https://github.com/adrienverge/yamllint) and [yamale](https://github.com/23andMe/Yamale). If not specified, these files are search in the current directory, the .ct directory in current directory, `$HOME/.ct`, and `/etc/ct`, in that order.

stevehipwell replied:
I strongly disagree, these files are primarily tool scoped and the other installation methods make sure they exist. Without these files the tool doesn't work without significant user interaction.

stevehipwell commented:
/verify
